test/integration/manifests/cilium/v1.13/cilium-operator/templates/deployment.yaml

```diff
@@ -152,6 +152,10 @@ spec:
       tolerations:
       - key: "CriticalAddonsOnly"
         operator: "Exists"
+      - operator: "Exists"
```

Comment:
We need these changes in private fork too ?

Reply:
yes, I can open a PR there as well